db/migrate/20191021130249_remove_schools_school_profiles_show_candidate_requirements_selection.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveSchoolsSchoolProfilesShowCandidateRequirementsSelection < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
Removing a column can impact existing running code - the active record model will cache columns at boot, then it will potentially try to write to a no longer existing column.

Instead this PR should remove usage of the column

```
class Schools::SchoolProfile < ApplicationRecord
  self.ignored_columns = %w(show_candidate_requirements_selection)
end
```

And a subsequent PR which gets released in a separate release should remove the database column